app/presenters/waste_exemptions_engine/certificate_presenter.rb

```diff
@@ -2,53 +2,26 @@
 
 module WasteExemptionsEngine
   class CertificatePresenter < BasePresenter
-
-    LOCALES_KEY = ".waste_exemptions_engine.pdfs.certificate"
-
-    # By complex we mean is there a need to display extra detail in the
-    # document. If its a partnership we display we display an extra section,
-    # which is a list of their names.
-    def complex_organisation_details?
-      return false unless %w[partnership].include?(business_type)
-
-      true
-    end
-
-    # The certificate displays headings on the left, and values from the
-    # registration on the right. Because this heading is dynamic based on the
-    # business type, we have a method for it in the presenter
-    def complex_organisation_heading
-      return I18n.t("#{LOCALES_KEY}.operator.partners") if business_type == "partnership"
-
-      I18n.t("#{LOCALES_KEY}.operator.business_name_if_applicable")
+    def partnership?
+      business_type == "Partnership"
     end
 
-    def complex_organisation_name
-      return operator_name unless business_type == "partnership"
-
-      # Based on the logic found in the existing certificate, we simply display
-      # the company name field unless its a partnership, in which case we list
-      # out all the partners
-      list_people
+    def partners_names
+      people.select(&:partner?).map do |person|
+        format("%<first>s %<last>s", first: person.first_name, last: person.last_name)
+      end.join("</br>").html_safe
     end
 
-    def expires_after_pluralized
-      ActionController::Base.helpers.pluralize(
-        WasteExemptionsEngine.configuration.years_before_expiry,
-        I18n.t("#{LOCALES_KEY}.changes.year")
-      )
+    def applicant_name
+      format("%<first>s %<last>s", first: applicant_first_name, last: applicant_last_name)
```

Comment:
Can we do any refactoring to reduce the duplicated `format("%<first>s %<last>s", first: x_first_name, last: x_last_name)`, since all 3 uses are very similar aside from the source of the names?